engine/src/test/java/com/datatorrent/stram/engine/StreamingContainerTest.java

```diff
@@ -94,4 +121,33 @@ public void emitTuples()
     private static final Logger logger = LoggerFactory.getLogger(CommitAwareOperator.class);
   }
 
+  private static class CommitAwareInputOperator extends CommitAwareOperator
+  {
+    public final transient DefaultOutputPort<String> output = new DefaultOutputPort<String>();
+  }
+
+  private static class CommitAwareProcessOperator extends CommitAwareOperator
+  {
+    private static final Logger logger = LoggerFactory.getLogger(CommitAwareOperator.class);
+    public static ArrayList<Long> committedWindowIds;
```

Comment:
I was referring to `public static ArrayList<Long> committedWindowIds;` not to `private static class CommitAwareProcessOperator`. Operator class does need to be static. I don't think it is necessary to have different loggers for every CommitAwareOperator. Please move logger to the StreamingContainerTest class and use it in CommitAwareOperator(s) if necessary.